cropper/src/main/java/com/canhub/cropper/CropWindowMoveHandler.java

```diff
@@ -20,7 +20,7 @@
  * Handler to update crop window edges by the move type - Horizontal, Vertical, Corner or Center.
  * <br>
  */
-final class CropWindowMoveHandler {
+public final class CropWindowMoveHandler {
```

Comment:
no issue, until we move all to kotlin we will have some issues like this.

As soon everything is in kotlin we will be able to make some public interfaces and internal implementations